### DISCUSSION TITLE
I am trying to use tailwind with a html file with no success, any help??

### DISCUSSION THREAD

**Discussion by joshuagarrido** [tailwindlabs/tailwindcss#12792] [Help] [answered]
![Screenshot 2024-01-20 160452](https://github.com/tailwindlabs/tailwindcss/assets/81715183/007c8f8c-cb10-4ce2-978d-76653b8477a8)
![Screenshot 2024-01-20 160527](https://github.com/tailwindlabs/tailwindcss/assets/81715183/6ecd2978-118e-42a3-86a3-8aeb23b29198)
![Screenshot 2024-01-20 160540](https://github.com/tailwindlabs/tailwindcss/assets/81715183/09826194-553a-4b55-b067-7999bb1da23c)
![Screenshot 2024-01-20 160553](https://github.com/tailwindlabs/tailwindcss/assets/81715183/ed5d1657-f346-4155-871b-739da77df463)
![Screenshot 2024-01-20 160557](https://github.com/tailwindlabs/tailwindcss/assets/81715183/8c6642cc-b9c9-4364-9358-1e7554f140f7)
![Screenshot 2024-01-20 160638](https://github.com/tailwindlabs/tailwindcss/assets/81715183/71c375a9-44a3-4448-954b-bce3324e0d13)
![Screenshot 2024-01-20 161647](https://github.com/tailwindlabs/tailwindcss/assets/81715183/320f6f7f-d0de-4005-889a-807977013cd6)

**Comment by wongjn** [comment, marked as answer]
For linking CSS with the `<link>` tag, the `rel` value should be [`stylesheet`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel#stylesheet), not `style_sheet`.

**Reply by joshuagarrido** [reply]
thank you!!! idk why i didn't see that 😳
